data/human/human missions.txt

```diff
@@ -7455,3 +7455,573 @@ mission "Adelita 5"
+					goto next2
+				`	"You have two offices?"`
+					goto offices
+				`	"I don't care about prying eyes! What do you want and what do I get?"`
+			`	Howard raises an eyebrow and then waves you off with a chuckle. "All in good time, my friend," he says, leaning back. "It's nothing you can't handle. I'll fill you in once we're safely on Sunracer, and let's just say it'll be worth your while."`
+				goto next2
+			label offices
+			`	Howard leans back with a knowing smile. "Well, 'office' is a bit generous for this place," he admits, gesturing around the room. "This is more of a glorified conference room I can reserve for investor meetings here on Hephaestus. My real office is back on Sunracer, where the serious deals happen. You'll see what I mean when we get there."`
+			choice
+				`	"Can't wait."`
+					goto next2
+				`	"Is that a glorified conference room too?"`
+			`	Howard's smile falters slightly. He forces a light chuckle, though the tension is noticeable. "Well, if I'm being honest, the Sunracer office is... also technically reservable," he admits, rubbing his neck and looking almost pained. "But just as nice! And, uh... look, my real office is technically a cubicle, but compared to the kiosk last time we met? It's a big step up." He shrugs, trying to brush it off. "But hey, we don't need fancy spaces to make serious deals. Let's get back on track, yeah?" He clears his throat.`
+			label next2
+			`	Howard's attention shifts away from you as he begins taking a series of work calls, his tone shifting seamlessly from charming to authoritative as he negotiates and makes deals. As he walks off he turns and mouths two words to you: "Spaceport... Tomorrow..." It's clear that, for now, Howard is finished with you and you'll get no more information.`
+
+mission "Syndicate Business 4"
+	name "Escort Howard to <planet>"
+	description "Escort Howard from Megaparsec Inc. to <planet> to learn what favor is needed in exchange for an investment opportunity."
+	source "Hephaestus"
+	to offer
+		has "Syndicate Business 3: done"
+	destination "Sunracer"
+	on offer
+		conversation
+			`You meet Howard at the spaceport. He's standing on a nearby landing pad by his Shuttle, dressed sharply but looking a bit hurried. He gives you a quick nod as you approach.`
+			`	"Good to see you're on time," Howard says, glancing at his antique wristwatch. "We've got a lot of ground--or space--to cover." Howard briefly looks at the <ship>. "Your <model> ought to go fast enough, I suppose. I'll be tied up in calls for most of the trip, but once we're back at the office, we can talk." He starts toward his shuttle's ramp, already shifting his attention back to his wristwatch. "Let's get moving."`
+				launch
+	npc accompany save
+		government "Syndicate"
+		personality escort timid
+		ship "Shuttle" "Gallant Knight"
+		on kill
+			conversation
+				`You watch as Howard's shuttle explodes into a cloud of debris along with any hope of that investment opportunity.`
```

Comment:
Oops...

Also maybe do this, because you don't know exactly _what_ investment opportunity.
```suggestion
				`You watch as Howard's shuttle explodes into a cloud of debris along with any hope of an investment opportunity.`
```